app/Http/Controllers/MissingController.php

```diff
@@ -8,16 +8,16 @@
 class MissingController extends Controller
 {
     //Spiele mit fehlenden Screenshots anzeigen
-    public function index_gamescreens(){
+    public function index_gamescreens() {
```

Comment:
__[code_sniffer]__
* Method name "MissingController::index_gamescreens" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->